lib/ansible/modules/network/meraki/meraki_network.py

```diff
@@ -172,7 +197,7 @@ def main():
     argument_spec = meraki_argument_spec()
     argument_spec.update(
         net_id=dict(type='str'),
-        type=dict(type='str', choices=['wireless', 'switch', 'appliance', 'combined'], aliases=['net_type']),
+        type=dict(type='list', choices=['wireless', 'switch', 'appliance'], aliases=['net_type']),
         tags=dict(type='str'),
```

Comment:
Why is tags not defined as a list ?

Reply:
I've decided I will update this to be a list. Again, break compatibility now.

Reply:
This does not break compatibility. Moving from a list to a string does break compatibility, but not this.